src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -862,6 +862,41 @@ async def SendCommand(self, nodeid: int, endpoint: int, payload: ClusterObjects.
             interactionTimeoutMs=interactionTimeoutMs, busyWaitMs=busyWaitMs, suppressResponse=suppressResponse).raise_on_error()
         return await future
 
+    async def SendBatchCommands(self, nodeid: int, commands: typing.List[typing.Tuple[int, ClusterObjects.ClusterCommand, typing.Type]],
+                                timedRequestTimeoutMs: typing.Union[None, int] = None,
```

Comment:
For batch commands - is this required then if ANY command in the set requires a timed invoke?

Reply:
kk, looks like it's checked below. You might have covered this already, but we should have a test for sending a batch of commands without a timed invoke when only one command in the batch requires it.

Also maybe a test for sending > 1 timed request command in a batch, to make sure a single timed request applies to all the commands in the batch.

Reply:
Mind if these test start rolling in next week? That is when I will have more integration tests and would be a good time for that extra command to make it's way in here